### ISSUE TITLE
Investigate System.Security.Claims added to fix compilation

### ISSUE THREAD

**Issue by pranavkm** [aspnet/Entropy#210]
https://github.com/aspnet/Entropy/commit/ce398e0bd91a86e3408693d7c197f3328215b612

**Comment by Eilon** [comment]
@HaoK @Tratcher any idea why this might be needed? Is something missing from NETStandard.Library or something?

**Comment by Tratcher** [comment]
No idea.

**Comment by HaoK** [comment]
Not sure, its also affecting identity I think, https://github.com/aspnet/Identity/issues/1141

**Comment by pranavkm** [comment]
I was able to remove it from Identity and it just built fine  - see https://github.com/aspnet/Identity/pull/1148. Like I said earlier, it was likely an artifact of having a mix of `netstandard1.x`, `net45x` and `net46`. I think it might be worth waiting until we make the jump to `netstandard2.0` before we clean these out (just in case we need to temporarily put these back before we have migrated everything).
